build.gradle.kts

```diff
@@ -380,11 +404,29 @@ publishing {
             artifact(sourcesJar.get()) {
                 classifier = "sources"
             }
+            // TODO: we probably will have to configure and add javadoc, as it seemingly required by Maven Central
+            // artifact(tasks.javadoc)
```

Comment:
Please note that we need to keep the following code in `build.gradle.kts`:

```
    replace("jvmSourcesJar", Jar::class).run {
        from(sourceSets["main"].allSource)
        // Also collect sources for the injected classes to simplify debugging
        from(project(":bootstrap").file("src"))
    }
```

Reply:
I'm afraid we need to add javadoc.

Reply:
> Please note that we need to keep the following code in build.gradle.kts

It is there, only the name of the task has changed to `sourcesJar`.

> I'm afraid we need to add javadoc.

Added it via Dokka.